src/SCSS/pages/_adminOnboarding.scss

```diff
@@ -161,6 +161,23 @@ progress::-moz-progress-bar {
 	width: 75%;
 }
 
+button {
```

Comment:
This will target every button in the app. I think it might be better to nest it or use a class instead.